docs/management/index-patterns.asciidoc

```diff
@@ -23,79 +18,93 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
 
 [float]
 [[settings-create-pattern]]
-== Creating an Index Pattern to Connect to Elasticsearch
-An _index pattern_ identifies one or more Elasticsearch indices that you want to explore with Kibana. Kibana looks for
-index names that match the specified pattern.
-An asterisk (*) in the pattern matches zero or more characters. For example, the pattern `myindex-*` matches all
-indices whose names start with `myindex-`, such as `myindex-1` and `myindex-2`.
+=== Creating an index pattern
 
-An index pattern can also simply be the name of a single index.
+To get started, go to *Management > Kibana > Index Patterns*.  Your index 
+patterns show up here, including any that were created when you added a sample data set. 
 
-To create an index pattern to connect to Elasticsearch:
+You can create a standard index pattern, or if a rollup index is detected in the 
+cluster, a rollup index pattern.
 
-. Go to the *Settings > Indices* tab.
-. Specify an index pattern that matches the name of one or more of your Elasticsearch indices. By default, Kibana
-guesses that you're working with log data being fed into Elasticsearch by Logstash.
-+
-NOTE: When you switch between top-level tabs, Kibana remembers where you were. For example, if you view a particular
-index pattern from the Settings tab, switch to the Discover tab, and then go back to the Settings tab, Kibana displays
-the index pattern you last looked at. To get to the create pattern form, click the *Add* button in the Index Patterns
-list.
+[role="screenshot"]
+image:management/index-patterns/images/rollup-index-pattern.png["Menu with rollup index pattern"]
 
-. If your index contains a timestamp field that you want to use to perform time-based comparisons, select the *Index
-contains time-based events* option and select the index field that contains the timestamp. Kibana reads the index
-mapping to list all of the fields that contain a timestamp.
+[float]
+==== Standard index pattern
 
-. Click *Create* to add the index pattern.
+{kib} makes it easy for you to create an index pattern by walking you through 
+the process. Just start typing in the *Index pattern* field and {kib} looks for 
+the names of {es} indices that match your input. If you want to include 
+system indices in your search, toggle the switch in the upper right.
 
-. To designate the new pattern as the default pattern to load when you view the Discover tab, click the *favorite*
-button.
+[role="screenshot"]
+image:management/index-patterns/images/create-index-pattern.png["Create index pattern"]
 
-[float]
-[[set-default-pattern]]
-== Setting the Default Index Pattern
-The default index pattern is loaded automatically when you view the *Discover* tab. Kibana displays a star to the
-left of the name of the default pattern in the Index Patterns list on the *Settings > Indices* tab. The first pattern
-you create is automatically designated as the default pattern.
+You can match multiple indices to create a combination index pattern.  
```

Comment:
Not sure `combination index pattern` makes sense. Maybe just "Your index pattern can match multiple {es} indices allowing you to group indices into a single index pattern" or something along those lines